frontend/src/components/noticeOfWork/NoticeOfWorkApplication.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
+import { Button, Collapse, Icon } from "antd";
+import NOWGeneralInfo from "@/components/noticeOfWork/NOWGeneralInfo";
+import NOWWorkPlan from "@/components/noticeOfWork/NOWWorkPlan";
+import { AuthorizationGuard } from "@/HOC/AuthorizationGuard";
+/**
+ * @class NoticeOfWorkApplication - contains all information regarding to a notice of work application
+ */
+
+const { Panel } = Collapse;
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
What's the reason for needing this?

Reply:
This component will be stateful once I start working with the data

Reply:
Gotcha 👍